index.html

```diff
@@ -375,12 +376,103 @@ <h3 style="margin-bottom: 100px">Terminal</h3>
             </p>
           </div>
         </div>
+      </section>
+
+      <section class="form-section">
+        <form action="https://formspree.io/f/{form_id}" method="POST">
+          <ul>
+            <li>
+              <input
+                type="text"
+                class="form-type"
+                name="full_name"
+                maxlength="30"
+                placeholder="Full Name"
+                required
+              />
+            </li>
 
-        <!-- <div class="about-me-text-container"></div> -->
+            <li>
+              <input
+                type="email"
+                class="form-type"
+                name="email"
+                placeholder="Email Address"
+                required
+              />
+            </li>
+
+            <li>
+              <textarea
+                name="message"
+                class="textarea"
+                maxlength="500"
+                placeholder="Enter Text Here"
+                rows="5"
+                cols="20"
+                required
+              ></textarea>
+            </li>
+
+            <li class="button">
+              <button type="submit" class="button-text">Get in touch</button>
+            </li>
```

Comment:
- [ ] Good job working on your submit button. You seem to have left out the button interactive states i.e on hover, active etc. Kindly include them. Below I will include the interactive states as provided in Figma

![image](https://user-images.githubusercontent.com/86318284/199227925-2ed46173-0cea-45da-a0dc-650211633d98.png)

Reply:
Thanks so much for the suggestion. I have resolved this also. 
I applied the interactivity below to the button.

`.button: hover {
    border-radius: 10px;
    opacity: 0.5;
  }

  .button:active {
    background: #ff6b00;
  }

  .button:visited {
    background-color: #eb9260;
  }

  .button:disabled {
    background-color: #979493;
  }
`